loadtest/user/user.go

```diff
@@ -34,13 +34,16 @@ type User interface {
 	// user
 	GetMe() (string, error)
 	GetPreferences() error
+	UpdatePreferences(pref *model.Preferences) error
 	CreateUser(user *model.User) (string, error)
 	UpdateUser(user *model.User) error
 	PatchUser(userId string, patch *model.UserPatch) error
 	GetUsersByIds(userIds []string) ([]string, error)
 	GetUsersByUsernames(usernames []string) ([]string, error)
 	GetUserStatus() error
 	GetUsersStatusesByIds(userIds []string) error
+	GetUsersInChannel(channelId string, page, perPage int) error
```

Comment:
Nit: I think we should add comments here, from the function name I would expect to return `users`. Just to prevent confusion.

Reply:
We should document all these functions at some point, probably a separate PR focusing on documentation.
Also, the initial idea was to actually never return anything except for an error and let the data go directly into the `UserStore`.
We might want to revisit this during some refactoring session to make sure we keep consistency when possible. I'll create a ticket for this.